1.1/geo.ttl

```diff
@@ -12,41 +12,63 @@
 
 geosparql: 
 	a owl:Ontology ;
-  dcterms:title "GeoSPARQL Ontology" ;
+  	dcterms:title "GeoSPARQL Ontology" ;
 	dcterms:publisher [
-	 a sdo:Organization ;
-	 sdo:name "Open Geospatial Consortium" ;
-	 sdo:url "https://www.ogc.org"^^xsd:anyURI ;
+		a sdo:Organization ;
+		sdo:name "Open Geospatial Consortium" ;
+		sdo:url "https://www.ogc.org"^^xsd:anyURI ;
 	] ;
 	dcterms:creator "OGC GeoSPARQL Standards Working Group" ;
-	dcterms:contributor [
+	dcterms:contributor 
+	[
+		a sdo:Person ;
+		sdo:name "Matthew Perry" ;
+		sdo:affiliation "Oracle Corporation" ;
+		sdo:email "matthew.perry@oracle.com"^^xsd:anyURI ;
+		sdo:url "?????????"^^xsd:anyURI ;
+	] ,
+	[
+		a sdo:Person ;
+		sdo:name "John Herring" ;
+		sdo:affiliation "Oracle Corporation" ;
+		sdo:email "john.herring@oracle.com"^^xsd:anyURI ;
+		sdo:url "?????????"^^xsd:anyURI ;
+	] ,
+	[
 		a sdo:Person ;
 		sdo:name "Nicholas J. Car" ;
 		sdo:affiliation "SURROUND Australia Pty Ltd" ;
-		sdo:email "nicholas.car@surroundaustralia.com" ;
-		sdo:url <https://orcid.org/0000-0002-8742-7730> ;
+		sdo:email "nicholas.car@surroundaustralia.com"^^xsd:anyURI ;
+		sdo:url "https://orcid.org/0000-0002-8742-7730"^^xsd:anyURI ;
 	] ,
 	[
 		a sdo:Person ;
 		sdo:name "Joseph Abhayaratna" ;		
 		sdo:affiliation "Geoscape Australia" ;
-		sdo:email "joseph.abhayaratna@geoscape.com.au" ;
-		sdo:url <https://orcid.org/0000-0001-5075-6234> ;
+		sdo:email "joseph.abhayaratna@geoscape.com.au"^^xsd:anyURI ;
+		sdo:url "https://orcid.org/0000-0001-5075-6234"^^xsd:anyURI ;
 	],
 	[
 		a sdo:Person ;
 		sdo:name "Timo Homburg" ;		
 		sdo:affiliation "Mainz University Of Applied Sciences" ;
-		sdo:email "timo.homburg@hs-mainz.de" ;
-		sdo:url <https://orcid.org/0000-0002-9499-5840> ;
+		sdo:email "timo.homburg@hs-mainz.de"^^xsd:anyURI ;
+		sdo:url "https://orcid.org/0000-0002-9499-5840"^^xsd:anyURI ;
 	],
 	[
 		a sdo:Person ;
 		sdo:name "Simon J D Cox" ;		
 		sdo:affiliation "CSIRO Australia" ;
-		sdo:email "simon.cox@csiro.au" ;
-		sdo:url <https://orcid.org/0000-0002-3884-3420> ;
-	] ;
+		sdo:email "simon.cox@csiro.au"^^xsd:anyURI ;
+		sdo:url "https://orcid.org/0000-0002-3884-3420"^^xsd:anyURI ;
+	] ,
+	[
+		a sdo:Person ;
+		sdo:name "Frans Knibbe" ;		
+		sdo:affiliation "CSIRO Australia" ;
```

Comment:
This line should be removed.

Reply:
@FransKnibbe I think your affiliation should be Geodan, correct?

Reply:
Not any longer, @kjano. I went into self-isolation before it became fashionable. For the moment I am unaffiliated; pursuing a few private interests.

Reply:
I've updated this